app/Album.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Album extends Model
+{
+    public function Reciters()
+    {
+        return $this->belongsTo('App\Reciter', 'reciters_id', 'id');
```

Comment:
For `'App\Reciter'`, I'd suggest using `Reciter::class` without the quotes. This would require adding `use App\Reciter;` to the top of the file.